jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -407,29 +407,30 @@ def test_parse_hst_drz(self, imviz_helper):
         assert_allclose(sky.dec.deg, 10.802045612042956, rtol=1e-3)
         data_unit = u.electron / u.s
         assert_quantity_allclose(tbl[0]['background'], 0.0014 * data_unit)
-        assert_quantity_allclose(tbl[0]['sum'], 126.63604794 * data_unit, rtol=1e-3)
+        assert_quantity_allclose(tbl[0]['sum'], 126.219763 * data_unit, rtol=1e-3)
         assert_quantity_allclose(tbl[0]['sum_aper_area'], 2583.959958 * (u.pix * u.pix), rtol=1e-3)
         assert_array_equal(tbl[0]['pixarea_tot'], None)
         assert_array_equal(tbl[0]['aperture_sum_counts'], None)
         assert_array_equal(tbl[0]['aperture_sum_counts_err'], None)
         assert_array_equal(tbl[0]['counts_fac'], None)
         assert_array_equal(tbl[0]['aperture_sum_mag'], None)
         assert_array_equal(tbl[0]['flux_scaling'], None)
-        assert_quantity_allclose(tbl[0]['min'], -0.02858124 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['max'], 3.94278287 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mean'], 0.04910025 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['median'], 0.02168917 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mode'], -0.033133 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['std'], 0.144362 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['mad_std'], 0.0233135 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['var'], 0.02084 * (data_unit * data_unit), rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['biweight_location'], 0.02098 * data_unit, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['biweight_midvariance'], 0.00062222 * (data_unit * data_unit), rtol=1e-3)  # noqa
-        assert_quantity_allclose(tbl[0]['fwhm'], 25.22627 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['semimajor_sigma'], 12.9581226 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['semiminor_sigma'], 7.8490214 * u.pix, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['orientation'], 88.733606 * u.deg, rtol=1e-3)
-        assert_quantity_allclose(tbl[0]['eccentricity'], 0.795676, rtol=1e-3)
```

Comment:
Is this related to a change in an upstream package? This test uses a subset but it's a spatial one in Imviz and that should not be effected by the changes in this PR.

Reply:
Is this some rebase gone wrong? Why are you changing the expected results here?

The remote data test suite is passing in `main`.

Reply:
This was most likely a caching mistake.